packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js

```diff
@@ -49,6 +49,7 @@ function PieWidgetUI({
   data = [],
   formatter,
   tooltipFormatter,
+  percentFormatter,
```

Comment:
Does this work also in `ComparativePieWidget`?

Reply:
No, I didn't implement it in ComparativePieWidget, I could implement this feature in comparative mode, but maybe could be far away of the scope of this feature

Reply:
ComparativePie Widget was indeed coming from PS, if I'm not wrong